ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,527 @@
+following these steps should keep the Gentoo repository in a clean working
+state, keep your activity up and ease your tasks as a maintainer.
+</p>
+
+<p>
+Your general responsibilities as a maintainer consist of keeping your packages
+up-to-date with upstream releases, responding to and fixing bugs assigned to
+you, and handling possible stabilization of your packages in time. You also
+need to keep up with the Python, Ruby, PHP, etc. ecosystems in Gentoo.
+</p>
+
+<p>
+<e>Rationale: If your package has existing stable keywords you <b>must</b>
+request stabilization of a newer version within a reasonable timeframe (usually
+at least 30 days, but see the
+<uri link="::keywording/#Moving from ~arch to arch">stabilization section
+</uri>), provided there are no regressions compared to the last stable version.
+This prevents stable users from being stuck on old versions of the package with
+no justification.</e>
+</p>
+
+<p>
+<e>Rationale: It's also important to add the latest e.g. Python versions which
+your package supports as soon as possible to avoid conflicts for users when the
+default Python target in the Gentoo profiles changes.</e> See
+<uri link="::ebuild-maintenance/general/#Daily / Occassional pkgcheck scans"/>for
+a tip to quickly check for a PYTHON_COMPAT update in your packages.
+</p>
+
+<p>
+If you're looking for tips on how to be aware of tasks available to you as a
+maintainer, please see the
+<uri link="::ebuild-maintenance/general/#External tools"/> section below.
+</p>
+
+</body>
+
+<subsection>
+<title>Bugzilla</title>
+<body>
+
+<p>
+<uri link="https://bugs.gentoo.org/">Gentoo's Bugzilla</uri> is still the main
+communication channel to reach you with your package state. When a bug report
+gets assigned to you, the first thing you should attempt to do is to confirm
+and replicate it with the information attached to the bug. You should then
+either mark the bug as "CONFIRMED", or provide a comment that you can't.
+</p>
+
+<p>
+Keep an eye out for CI/tinderbox-style bugs too, which are found by testing the
+latest ebuilds (by commit) and random packages, respectively. They reveal
+breakages on different systems, mostly missing dependencies, and make you face
+issues you perhaps hadn't thought of before.
+</p>
+
+<p>
+When you make a commit to Gentoo's git tree, you can reference or close a
+single bug, or multiple bugs, by using <e>Bug:</e> or <e>Closes:</e> tag in
+your git commit message's footer area.
+<uri link="::ebuild-maintenance/git/#Git commit message format">See an example,
+and more documentation</uri>.
+</p>
+
+<p>
+<uri link="::keywording/#Stabilization rules">Stabilization can be
+done by a maintainer who is also a developer, and owns relevant hardware</uri>.
+It can also be <uri link="::keywording/#Moving from ~arch to arch">
+requested through Bugzilla</uri> for arch teams to handle, even for your own
+packages.
+</p>
+
+<p>
+It is recommended to set up your mail client's filtering properly. Separating
```

Comment:
Maybe we should link to [this](https://wiki.gentoo.org/wiki/User:Wraeth/Sieve) or [this](https://wiki.gentoo.org/wiki/User:Wraeth/Sieve)?

Reply:
You gave the same link twice - but I guess https://wiki.gentoo.org/wiki/Project:Infrastructure/Developer_E-Mail/Sieve_Example would be the 2nd one? Yeah, a link to a **good** mail filter setup would be great.

Reply:
Sorry, yes!